hack/jenkins/upload-to-gcs.sh

```diff
@@ -85,7 +85,8 @@ function upload_version() {
     gsutil -q -h "Content-Type:application/json" cp -a "${gcs_acl}" <(
       echo "{"
       echo "    \"version\": \"${version}\","
-      echo "    \"timestamp\": ${timestamp}"
+      echo "    \"timestamp\": ${timestamp},"
+      echo "    \"node\": \"${NODE_NAME:-}\""
```

Comment:
"node" is super-overloaded in our codebase. it's *probably* clear this isn't referring to a k8s node, but maybe name this `jenkins-node` instead?